drivers/eeprom/eeprom_at2x.c

```diff
@@ -201,29 +209,58 @@ static size_t eeprom_at2x_size(const struct device *dev)
 }
 
 #ifdef CONFIG_EEPROM_AT24
+
+static uint16_t eeprom_at24_translate_offset(const struct device *dev,
+					     off_t *offset)
+{
+	const struct eeprom_at2x_config *config = dev->config;
+
+	const uint16_t addr_incr = *offset >> config->addr_width;
+	*offset &= BIT_MASK(config->addr_width);
+
+	return config->bus_addr + addr_incr;
+}
+
+static size_t eeprom_at24_adjust_read_count(const struct device *dev,
+					    off_t offset, size_t len)
+{
+	const struct eeprom_at2x_config *config = dev->config;
+	const size_t remainder = BIT(config->addr_width) - offset;
+
+	if (len > remainder)
+		len = remainder;
```

Comment:
braces must be added.
> Add braces to every if, else, do, while, for and switch body, even for single-line code blocks.

Reply:
Check path report the following if I do so

WARNING: braces {} are not necessary for single statement blocks
#71: FILE: drivers/eeprom/eeprom_at2x.c:237:
+	if (len > remainder) {
+		len = remainder;
+	}